### PR TITLE
README.md, src/doc/en/installation: Replace release tours links

### PR DESCRIPTION
**Pull request by mkoeppe**
The release tours are now accessed through GitHub Releases.

This is the next step of:
- #31533 

### :memo: Checklist

- [x] The title is concise and informative.
- [x] The description explains in detail what this PR is about.
- [x] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation and checked the documentation preview.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -489,8 +489,8 @@ Troubleshooting
 ---------------
 
 If you have problems building Sage, check the Sage Installation Guide,
-as well as the version-specific Sage Installation FAQ in the [Sage Release
-Tour](https://wiki.sagemath.org/ReleaseTours) corresponding to the
+as well as the version-specific Sage Installation FAQ in the [release
+tour](https://github.com/sagemath/sage/releases) corresponding to the
 version that you are installing.
```
Comment (kwankyu) [inline README.md:494]: There is no "Installation FAQ". How about "the version-specific installation help in the ..."?

File: src/doc/en/installation/troubles.rst [hunk 2]
```diff
@@ -8,8 +8,8 @@ the :ref:`sec-installation-from-sources` or use one of the alternatives
 proposed at the end of :ref:`installation-guide`.
 
 If you have any problems building or running Sage, please take a look
-at the Installation FAQ in the `Sage Release Tour
-<https://wiki.sagemath.org/ReleaseTours>`_ corresponding to the version
+at the Installation FAQ in the `release tour
+<https://github.com/sagemath/sage/releases>`_ corresponding to the version
 that you are installing.  It may offer version-specific installation
```
Comment (kwankyu) [inline src/doc/en/installation/troubles.rst:13]: "at the installation help in the"?

**Review by kwankyu — APPROVED** [review]
LGTM.

**Comment by mkoeppe** [comment]
Thank you!

**Pull request merged**